addons/mail/static/tests/new/mobile/mobile_tests.js

```diff
@@ -0,0 +1,35 @@
+/** @odoo-module **/
+
+import { patchUiSize } from "@mail/../tests/helpers/patch_ui_size";
+import { click, start, startServer } from "@mail/../tests/helpers/test_utils";
+import { getFixture } from "@web/../tests/helpers/utils";
+
+let target;
+QUnit.module("mobile", {
+    async beforeEach() {
+        target = getFixture();
+    },
+});
+
+QUnit.test(
+    "auto-select 'Inbox' when discuss had channel as active thread",
```

Comment:
The test used to be on a mobile specific test suite (see manifest), isn't it running on the normal suite now?

Reply:
Yes it is running on the normal one. Do we still need to split?